components/BookDetailsPage/BookJsonLd.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2018 GDL
+ *
+ * See LICENSE
+ */
+import React from 'react';
+import type { BookDetails } from '../../types';
+
+type Props = {|
+  book: BookDetails
+|};
+
+export default function BookJsonLd({ book }: Props) {
+  // Filter away empty strings
+  const cleanedContributors = book.contributors.filter(c => c.name);
+
+  const authors = cleanedContributors
+    .filter(c => c.type === 'Author') // Check if we have a name. In the dataset this is blank sometimes
```

Comment:
Ser at det i `types.js` defineres 
```
export type Contributor = {
  id: number,
  name: string
  name: string,
  type:
    | 'Author'
    | 'Illustrator'
    | 'Translator'
    | 'Photographer'
    | 'Contributor'
};
```
Kan denne brukes til filtreringen i stedet for å hardkode f.eks `Author`?